InferenceSystem/deploy-aci.yaml

```diff
@@ -4,61 +4,78 @@ identity: null
 location: westus2
 name: live-inference-system-aci-allhydrophones-3gb-new
 properties:
-  containers:
-  - name: orcasound-lab-live
-    properties:
-      environmentVariables:
-      - name: AZURE_COSMOSDB_PRIMARY_KEY
-        secureValue: '<cosmos_primary_key>'
-      - name: AZURE_STORAGE_CONNECTION_STRING
-        secureValue: '<storage_connection_string>'
-      - name: INFERENCESYSTEM_APPINSIGHTS_CONNECTION_STRING
-        secureValue: '<appinsights_connection_string>'
-      image: orcaconservancycr.azurecr.io/live-inference-system:11-15-20.FastAI.R1-12.OrcasoundLab.v0
-      ports: []
-      resources:
-        requests:
-          cpu: 1.0
-          memoryInGB: 3
-  - name: port-townsend-live
-    properties:
-      environmentVariables:
-      - name: AZURE_COSMOSDB_PRIMARY_KEY
-        secureValue: '<cosmos_primary_key>'
-      - name: AZURE_STORAGE_CONNECTION_STRING
-        secureValue: '<storage_connection_string>'
-      - name: INFERENCESYSTEM_APPINSIGHTS_CONNECTION_STRING
-        secureValue: '<appinsights_connection_string>'
-      image: orcaconservancycr.azurecr.io/live-inference-system:11-15-20.FastAI.R1-12.PortTownsend.v0
-      ports: []
-      resources:
-        requests:
-          cpu: 1.0
-          memoryInGB: 3
-  - name: bush-point-live
-    properties:
-      environmentVariables:
-      - name: AZURE_COSMOSDB_PRIMARY_KEY
-        secureValue: '<cosmos_primary_key>'
-      - name: AZURE_STORAGE_CONNECTION_STRING
-        secureValue: '<storage_connection_string>'
-      - name: INFERENCESYSTEM_APPINSIGHTS_CONNECTION_STRING
-        secureValue: '<appinsights_connection_string>'
-      image: orcaconservancycr.azurecr.io/live-inference-system:11-15-20.FastAI.R1-12.BushPoint.v0
-      ports: []
-      resources:
-        requests:
-          cpu: 1.0
-          memoryInGB: 3
-  imageRegistryCredentials:
-  - server: orcaconservancycr.azurecr.io
-    username: orcaconservancycr
-    password: '<image_registry_password>'
-  osType: Linux
-  restartPolicy: Always
+	containers:
```

Comment:
Can you change from tabs to whitespace for consistency?

Reply:
done